### PR TITLE
Action text stuff

### PR DESCRIPTION
**Pull request by Bod9001**
not tested in multiplayer

### CODE REVIEW

File: UnityProject/Assets/Scripts/UI/Core/ChatBubble/ActionText.cs [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+			Text.alpha = 1;
+		}
+
+		Text.alpha = flot;
+
+
+	}
+
+	[NaughtyAttributes.Button()]
+	public void Startsdasds()
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/UI/Core/ChatBubble/ActionText.cs]: wat

File: UnityProject/Assets/Scripts/UI/Core/ChatBubble/ActionText.cs [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+			{
+				LeanTween.cancel(this.gameObject);
+				DoFade = false;
+				Zero();
+			}
+		}
+	}
+
+	[NaughtyAttributes.Button()]
+	public void Zero()
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/UI/Core/ChatBubble/ActionText.cs]: can you rename this to something that describes what it does?

File: UnityProject/Assets/Scripts/UI/Core/ChatBubble/ChatBubbleManager.cs [hunk 3]
```diff
@@ -75,6 +77,60 @@ IEnumerator InitCache()
...
 
+
+	/// <summary>
+	/// Display a chat bubble and make it follow a transform target
+	/// </summary>
+	/// <param name="msg">Text to show in the Action</param>
+	/// <param name="followTarget">The transform in the world for the bubble to follow</param>
+	/// <param name="chatModifier">Any chat modifiers that need to be applied</param>
+	[NaughtyAttributes.Button]
+	public static void ShowAction(string msg)
```
Comment (corp-0) [inline UnityProject/Assets/Scripts/UI/Core/ChatBubble/ChatBubbleManager.cs:85]: why are the params on the docstring not part of the method params?

File: UnityProject/Assets/Scripts/UI/Core/ChatBubble/ActionText.cs (hunk 1 above)
Comment (BootlegBow) [reply UnityProject/Assets/Scripts/UI/Core/ChatBubble/ActionText.cs]: startsdasds

**Comment by Bod9001** [comment]
yeah, quickly done want to get working on addressable's

**Comment by Bod9001** [comment]
Done requested changes

**Pull request merged**